### PR TITLE
don't open a new https connection for every log flush

### PR DESCRIPTION
**Pull request by zt-initech**

### CODE REVIEW

**Comment by idohalevi** [comment]
@zt-initech Thank you! I'll go over it during the week.

File: logzio/sender.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 
 MAX_BULK_SIZE_IN_BYTES = 1 * 1024 * 1024  # 1 MB
+requests_session = requests.Session()
```
Comment (idohalevi) [inline logzio/sender.py]: @zt-initech 
Why did you opt to have requests.Session outside of LogzioHandler class instead of a member?

Comment (zt-initech) [reply logzio/sender.py]: I've changed it to me a member

**Comment by zt-initech** [comment]
@idohalevi ping?

**Comment by idohalevi** [comment]
> @idohalevi ping?

Sorry for that, I was out. I will go over it one more time today and release a new version

**Pull request merged**